modules/stanford_profile_styles/stanford_profile_styles.module

```diff
@@ -43,6 +43,17 @@ function stanford_profile_styles_page_attachments(array &$attachments) {
   }
 }
 
+/**
+ * Implements hook_library_info_alter().
+ */
+function stanford_profile_styles_library_info_alter(&$libraries, $extension) {
+  // Replace react_paragraphs field_formatter css with our own.
+  if ($extension == "react_paragraphs" && isset($libraries["field_formatter"])) {
+    unset($libraries['field_formatter']['css']['component']['js/build/css/react_paragraphs.field_formatter.css']);
+    $libraries['field_formatter']['dependencies'][] = "stanford_profile_styles/react_paragraphs";
+  }
+}
```

Comment:
This alter removes the css from the react_paragraphs module altogether and replaces it with ours. This way anywhere that module was going to add the css we new load ours and don't have to override their styles with more stronger css.

Reply:
That is helpful to keep us from writing stronger selectors.

Reply:
When we get into sub-theming this becomes much easier as you can do this in the theme's info.yml file.

https://www.drupal.org/docs/8/theming/adding-stylesheets-css-and-javascript-js-to-a-drupal-8-theme#override-extend

It is a super handy way of saying, "I don't want any of that, let me write my own thing and get out of the way."